src/rpc/blockchain.cpp

```diff
@@ -255,7 +255,7 @@ void RPCNotifyBlockChange(const CBlockIndex* pindex)
 static RPCHelpMan waitfornewblock()
 {
     return RPCHelpMan{"waitfornewblock",
-                "\nWaits for a specific new block and returns useful info about it.\n"
+                "\nWaits for any new block and returns useful info about it.\n"
```

Comment:
In commit "rpc: fix waitfornewblock description" (e2a1347db07933d1c6e40ab45f2ca4c56c691fed)

This is a good documentation fix, but as described previously this waitfornewblock interface is inherently racy. Ideally it would accept an `current_block` option or argument, and return as soon as the tip is not not pointing to that block. This could be something to add in a followup PR, though, because it is not directly relevant to this one.

Reply:
I made a note to either add that argument in this PR or make a followup.